docs/tutorials/index.md

```diff
@@ -38,6 +38,7 @@ Select API:&nbsp;
     * [Visual Question Answering](http://gluon.mxnet.io/chapter08_computer-vision/visual-question-answer.html) <img src="https://upload.wikimedia.org/wikipedia/commons/6/6a/External_link_font_awesome.svg" alt="External link" height="15px" style="margin: 0px 0px 3px 3px;"/>
 * Practitioner Guides
     * [Multi-GPU training](http://gluon.mxnet.io/chapter07_distributed-learning/multiple-gpus-gluon.html) <img src="https://upload.wikimedia.org/wikipedia/commons/6/6a/External_link_font_awesome.svg" alt="External link" height="15px" style="margin: 0px 0px 3px 3px;"/>
+    * [Distributed Training](https://github.com/apache/incubator-mxnet/tree/master/example/distributed_training)
```

Comment:
Not a big fan to link to the github repo. I understand the tutorial is not runnable but I would prefer if it was hosted in the tutorials/docs/gluon to avoid fragmentation. Add the right import statements, and put in 
```
runnable: 
    ```python 
    store = mxnet.kv.create('dist')
    ```
not meant to be run: 
    ```
     for batch in train_data:
         train_batch(batch, ctx, net, trainer)
    ```
```
That way the code you are using is tested against the CI for correctness.

Reply:
I don't think the CI system can currently test distributed training. I think you are trying to say part of the code can be tested. Note that it will only test very trivial parts and those parts will lose syntax highlighting which will spoils user experience.